server/webapp/WEB-INF/rails.new/app/assets/new_stylesheets/single_page_apps/pipeline_configs/_accordion.scss

```diff
@@ -14,18 +14,29 @@
  * limitations under the License.
  */
 
+$accordion-backg: #CFD8DC;
```

Comment:
This stylesheet is all new styles, not just variable overrides, hence created a new file.